recipes/libnetcdf/bld.bat

```diff
@@ -0,0 +1,23 @@
+mkdir %SRC_DIR%\build
+cd %SRC_DIR%\build
+
+set HDF5_DIR=%LIBRARY_PREFIX%\cmake
+
+if "%ARCH%" == "64" (
+  set CMAKE_GENERATOR="Visual Studio 10 2010 Win64"
+) else (
+  set CMAKE_GENERATOR="Visual Studio 10 2010"
+)
+
+cmake %SRC_DIR% -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%  -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DENABLE_TESTS=OFF -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% -G %CMAKE_GENERATOR%
+if errorlevel 1 exit 1
+
+cmake --build .
+if errorlevel 1 exit 1
+
+cmake --build . --target INSTALL
+if errorlevel 1 exit 1
+
+:: Remove msvc redist files.
+cd %LIBRARY_BIN%
+rm msvc*.dll
```

Comment:
it is weird to manually remove the dlls here.  Maybe the NetCDF install step copied them in?  If so, keep this removal.  Otherwise, this is cruft.